cpp/src/linear_programming/utilities/ping_pong_graph.cuh

```diff
@@ -60,13 +60,13 @@ class ping_pong_graph_t {
       if (total_pdlp_iterations % 2 == 0 && !even_initialized) {
         RAFT_CUDA_TRY(cudaStreamEndCapture(stream_view_.value(), &even_graph));
         // Extra NULL NULL 0 mandatory for cuda 11.8
```

Comment:
We can remove the cuda 11.8 comment as well.
Could you also update more occurances of `cudagraphInstantiate`:
- https://github.com/NVIDIA/cuopt/blob/branch-25.08/cpp/src/routing/cuda_graph.cuh#L57
- https://github.com/NVIDIA/cuopt/blob/branch-25.08/cpp/src/mip/presolve/load_balanced_bounds_presolve.cu#L206-L209
- https://github.com/NVIDIA/cuopt/blob/branch-25.08/cpp/src/mip/feasibility_jump/feasibility_jump.cu#L788